### PR TITLE
Create script to generate Darwin Core Archive + EML

### PR DESCRIPTION
**Pull request by peterdesmet**

### CODE REVIEW

File: src/frictionless_to_dwc.Rmd [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+```{r}
+eml <- movepub::write_eml(
+  doi = versioned_doi,
+  directory = dir,
+  contact = contact,
+  derived_paragraph = TRUE
+)
+```
+
+Process EML (TODO):
```
Comment (peterdesmet) [inline src/frictionless_to_dwc.Rmd:61]: @sannegovaert these steps should be completed

File: src/zenodo_to_gbif.Rmd [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+```
+
+Process EML:
+
+```{r}
+# Get description from Zenodo (with HTML, see https://github.com/inbo/movepub/issues/65)
+zenodo <- jsonlite::read_json(zenodo_url)
+description_full <- zenodo$metadata$description
+
+# ...
```
Comment (peterdesmet) [inline src/zenodo_to_gbif.Rmd:70]: @sannegovaert can you describe here what you are doing? Add more comments below if it is hard to explain in a single sentence.

Comment (sannegovaert) [reply src/zenodo_to_gbif.Rmd]: done!

Comment (peterdesmet) [reply src/zenodo_to_gbif.Rmd]: With `eml$dataset$abstract$para <- description` we are currently overwriting the derived paragraph. This should be fixed.

Comment (sannegovaert) [reply src/zenodo_to_gbif.Rmd]: fixed

**Pull request merged**